server/app/lib/alloy.js

```diff
@@ -37,3 +37,36 @@ export async function verifyAlloy(first_name, last_name, address, city, state, z
     return null
   }
 }
+
+/*
+ *
+ * fuzzyAlloy()
+ *
+ * A fuzzy search of Alloy
+ *
+ */
+export async function fuzzyAlloy(first_name, last_name, state, zip) {
+  let alloyKey = ov_config.alloy_key
+  let alloySecret = ov_config.alloy_secret
+
+  console.log(`[ALLOY] Calling Fuzzy API with: ${first_name} ${last_name} ${state} ${zip}`)
+
+  try {
+    let response
+    if (alloyKey && alloySecret) {
```

Comment:
if there is no alloyKey and alloySecret, will we be returning `response.data.total` but that will crash, right?

Reply:
i don't think there's a case where we wouldn't have the alloy secret and key currently